src/test/seccomp.c

```diff
@@ -131,12 +131,38 @@ static void* run_thread(__attribute__((unused)) void* p) {
   return NULL;
 }
 
+static void test_get_action_avail(void) {
+  // `SECCOMP_RET_ALLOW` is available since the first version of `SECCOMP_GET_ACTION_AVAIL`
+  uint32_t action = SECCOMP_RET_ALLOW;
+  int ret = syscall(RR_seccomp, SECCOMP_GET_ACTION_AVAIL, 0, &action);
+  assert(ret == EINVAL || ret == 0);
+}
+
+static void test_get_notif_sizes(void) {
+  struct {
+    uint16_t seccomp_notif;
+    uint16_t seccomp_notif_resp;
+    uint16_t seccomp_data;
+  } sizes;
+  int ret = syscall(RR_seccomp, SECCOMP_GET_NOTIF_SIZES, 0, &sizes);
+  assert(ret == EINVAL || ret == 0);
+  if (ret == 0) {
+    // These were the sizes when `SECCOMP_GET_NOTIF_SIZES` was first added.
+    assert(sizes.seccomp_notif >= 80);
+    assert(sizes.seccomp_notif_resp >= 24);
+    assert(sizes.seccomp_data >= 64);
```

Comment:
These should be test_assert.

Reply:
Huh, don't know why I missed that. I assume all the other added ones should be as well.
I did check that a failure here trigger a test failure at least...

Reply:
Done.